docs/quickstart.rst

```diff
@@ -141,7 +141,7 @@ While this page shows only screenshots, the actual dashboard is interactive.
 
 .. note::
 
-    The :code:`FairlearnDashboard` will move from Fairlearn to the
+    The :class:`.FairlearnDashboard` will move from Fairlearn to the
```

Comment:
recently @riedgar-ms confirmed we don't need the `.`

Reply:
I commented on this in a different comment. Sadly, it very much seems to make a difference.

Reply:
I've seen cases where it does, and cases where it doesn't :-/